cmd/mt-index-migrate/main.go

```diff
@@ -63,8 +63,13 @@ func main() {
 		log.Fatal(4, "failed to create cql session for destination cassandra. %s", err)
 	}
 
+	// read key(space|table) information from templates
+	var tomlstring = util.FileToString(*cassandraTemplate)
+	var conf = util.TomlCassandraToVar(tomlstring)
```

Comment:
the caller of util shouldn't need to know about toml specifics or about the 2 separate steps.
I suggest instead an api like:
```
tableSchema, err := util.ReadEntry(*cassandraTemplate, "table-schema")
if err != nil {
...
}
```

Reply:
makes sense, i'll change this.